.github/workflows/integration_test.yaml

```diff
@@ -5,5 +5,7 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@integration-module-matrix
```

Comment:
Do we really want to switch to this, rather than landing whatever's in here in the main branch?

Reply:
No, this is just temporary to make the integration tests working, I will switch to the main branch after the [changes landed](https://github.com/canonical/operator-workflows/pull/50) in `operator-workflow` and before merge this PR.